main.rb

```diff
@@ -1,6 +1,27 @@
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+app = App.new
+puts 'Welcome to School Library App!\n\n'
+
+loop do
+  puts 'Please choose an option by enterin a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+
+  choice = gets.chomp.to_i
+
+  case choice
+  when 1 then app.list_books
+  when 2 then app.list_persons
+  when 3 then app.create_person
+  when 4 then app.create_book
+  when 5 then app.create_rental
+  when 6 then app.list_rentals
+  when 7
+    puts 'Thank you for using this app!\n'
+    break
+  end
+end
```

Comment:
- Hey, while running your app, it throws this error below:


![image](https://github.com/Fombi-Favour/school-library-rb/assets/59397159/2b8ecfc3-d8da-4434-bb71-3a2169dd3a5a)


This is because you've not required the relative file. for example, `require_relative 'app'` so these files could be read.

*Also, I don't want to duplicate this review comment, Kindly check the rest of your files, and ensure you've required all the files where they need to so they can be accessible.*

- More on this, check your app.rb file, you should have something similar to this below, 

```
require_relative 'person'
require_relative 'student'
require_relative 'teacher'
require_relative 'book'
require_relative 'classroom'
require_relative 'rental'
```
> I hope this is more explanatory now so you understand properly.
> Note: check ALL YOUR FILES to see if they're all accessible to other files where they've been used.